enhanced_viral_dashboard.py

```
@@ -2481,6 +2410,166 @@ def get_realtime_data_service():
         """)
 
+# Enhanced System Status and Analytics Overview - Moved above footer
+st.markdown("---")
+
+# Check NLP service status
+def check_nlp_service_status():
+    """Check if NLP service is running"""
+    try:
+        import requests
+        response = requests.get("http://localhost:8000/health", timeout=2)
+        return response.status_code == 200
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)